velox/connectors/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_connector Connector.cpp)
+add_library(velox_connector Connector.cpp tests/ConnectorTest.cpp)
```

Comment:
You are adding a test dependency to the connector library ? 
Is there a reason why ?

Reply:
Thanks Krishna I'm removing it